libs/saksbehandling-common/src/main/kotlin/VedtakType.kt

```diff
@@ -1,7 +1,7 @@
 package no.nav.etterlatte.libs.common.vedtak
 
 enum class VedtakType(
-    val kanLoepe: Boolean,
```

Comment:
@sebassonav ser du har en kommentar på dette sett opp mot OPPHOER, men "vanligBehandling" (spesielt _vanlig_) sier da veeeldig lite? Hva er vanlig? Hva er i tilfelle en _uvanlig_ behandling? Noe som ikke skal oppstå..? 

Det var en diskusjon på slack i forkant av navngivningen her, men jeg klarer ikke å finne den igjen. @madsop-nav hva synes du?

Reply:
Ja, vi diskuterte jo litt før vi hamna på `kanLoepe`. https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/4688#discussion_r1587629753

Reply:
Synes for så vidt vanleg er greitt nok.